pyaedt/hfss.py

```diff
@@ -1987,42 +1861,43 @@ def create_voltage_source_from_objects(self, startobj, endobject, axisdir=0, sou
             self.logger.error("One or both objects doesn't exists. Check and retry")
             return False
         if self.solution_type in ["Modal", "Terminal", "Transient Network"]:
             sheet_name, point0, point1 = self.modeler._create_sheet_from_object_closest_edge(
-                startobj, endobject, axisdir, source_on_plane
+                assignment, reference, start_direction, source_on_plane
             )
-            sourcename = self._get_unique_source_name(sourcename, "Voltage")
-            return self.create_source_excitation(sheet_name, point0, point1, sourcename, sourcetype="Voltage")
+            name = self._get_unique_source_name(name, "Voltage")
+            return self.create_source_excitation(sheet_name, point0, point1, name, source_type="Voltage")
         return False  # pragma: no cover
 
-    @pyaedt_function_handler()
-    def create_current_source_from_objects(self, startobj, endobject, axisdir=0, sourcename=None, source_on_plane=True):
+    @pyaedt_function_handler(startobj="assignment", endobject="reference", sourcename="name", axisdir="start_direction")
+    def create_current_source_from_objects(
+        self, assignment, reference, start_direction=0, name=None, source_on_plane=True
+    ):
         """Create a current source taking the closest edges of two objects.
 
         Parameters
         ----------
-        startobj :
-            Starting object for the integration line.
-        endobject :
-            Ending object for the integration line.
-        axisdir : int or :class:`pyaedt.application.Analysis.Analysis.AxisDir`, optional
-            Position of the port. It should be one of the values for ``Application.AxisDir``,
-            which are: ``XNeg``, ``YNeg``, ``ZNeg``, ``XPos``, ``YPos``, and ``ZPos``.
-            The default is ``Application.AxisDir.XNeg``.
-        sourcename : str, optional
+        assignment : str or int or :class:`pyaedt.modeler.cad.object3d.Object3d`
+            First solid connected to the current source.
+        reference : str or int or :class:`pyaedt.modeler.cad.object3d.Object3d`
+            Second object connected to the current source.
+        start_direction : int or :class:`pyaedt.application.Analysis.Analysis.AxisDir`, optional
+            Start direction for the port location.
+            It should be one of the values for ``Application.AxisDir``, which are: ``XNeg``, ``YNeg``,
+            ``ZNeg``, ``XPos``, ``YPos``, and ``ZPos``.
+             The default is ``Application.AxisDir.XNeg``.
+        name : str, optional
             Name of the source. The default is ``None``.
         source_on_plane : bool, optional
-            Whether to create the source on the plane orthogonal to ``axisdir``. The default is ``True``.
+            Whether to create the source on the plane orthogonal to ``start_direction``. The default is ``True``.
```

Comment:
```suggestion
            Whether to create the source on the plane orthogonal to
            the start direction. The default is ``True``.
```
Avoid using code entities in brief descriptions unless necessary.